iaso/tests/test_entities.py

```diff
@@ -381,12 +382,12 @@ def test_export_entity(self):
         self.assertEquals(response.get("Content-Disposition"), "attachment; filename=NEW CLIENT_ENTITY.xlsx")
 
         # export specific entity type as xlsx
-        response = self.client.get(f"/api/entity/?entity_type_id={entity_type.pk}&xlsx=true/")
+        response = self.client.get(f"/api/entity/?entity_type_ids={entity_type.pk}&xlsx=true/")
```

Comment:
Do we also have tests that make sure it works if multiple values are passed in `entity_type_ids`?

Reply:
nope, my bad, i have added this option